easybuild/easyconfigs/b/Boost/Boost-1.66.0-GCCcore-6.4.0-serial.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Boost'
+version = '1.66.0'
+versionsuffix = '-serial'
```

Comment:
this Boost can still work in parallel with threading, right?
in that case the `-serial` suffix is misleading. maybe use `-no_mpi` instead?

Reply:
@smoors, another good suggestion so I'll push an update ASAP.